### PR TITLE
V 1.0

### PR DESCRIPTION
**Pull request by galyfray**
This first version includes :
- Documentation, contributing guidelines and a Readme.
- A first working version of action capable of creating a comment based on the content of a specified file.
- Some debugging and demonstration workflows.

### CODE REVIEW

File: .github/workflows/example.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+jobs:
+  comment:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Add a comment to the PR
+        uses: Correlatio-company/Pull-Request-Comments@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kilian-goetz) [inline .github/workflows/example.yml:20]: Could it's possible to rename token name by an more explicit name ?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: Add a comment to the PR
+        uses: Correlatio-company/Pull-Request-Comments@development
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kilian-goetz) [inline .github/workflows/test.yml:19]: same comment for token name

File: README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  comment:
+    permissions:
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Add a comment to the PR
+        uses: Correlatio-company/Pull-Request-Comments@master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (kilian-goetz) [inline README.md:25]: same comment for github token name

**Review by kilian-goetz — CHANGES_REQUESTED** [review]

File: .github/workflows/example.yml (hunk 1 above)
Comment (galyfray) [reply .github/workflows/example.yml:20]: Well. Some part can be renamed. For example, this line can be valid : 
```YAML
          GITHUB_TOKEN_FOR_ACTION: ${{ secrets.GITHUB_TOKEN }}
```

Yet as the other variable `secrets.GITHUB_TOKEN` cannot be renamed. For constancy's sake, I would not recommand changing the name of the variable. As is, it is clear that we only pass a variable to the action, and looking for the meaning of this variable is quite easy thanks to GitHub's extensive documentation on the subject. Change the name here would also require updating the python code, and might make understanding what's going on inside of it harder.

Comment (Beladric) [reply .github/workflows/example.yml:20]: Okay no problem. So rename `GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}` to `GITHUB_TOKEN_FOR_ACTION: ${{ secrets.GITHUB_TOKEN }}` and you can merge.

Comment (kilian-goetz) [reply .github/workflows/example.yml:20]: Okay no problem I understand. You can merge.

**Review by kilian-goetz — APPROVED** [review]

**Pull request merged**